VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts

```diff
@@ -120,14 +121,33 @@ describe('ParticipantsPanelComponent', () => {
     it('should not process eventhub endpoint updates not in list', () => {
         component.setupEventhubSubscribers();
         const status = EndpointStatus.InConsultation;
-        const ep = endpoints[0];
         const message = new EndpointStatusMessage(Guid.create().toString(), conferenceId, status);
 
         endpointStatusSubjectMock.next(message);
 
         expect(component.participants.find(x => x.id === message.endpointId)).toBeUndefined();
     });
 
+    it('should set transferring in when HearingTransfer In event received', () => {
+        const p = participants[0];
+        component.handleHearingTransferChange(new HearingTransfer(component.conferenceId, p.id, TransferDirection.In));
```

Comment:
I would recommend using `hearingTransferSubjectMock` to test the subscription to the event service
This can be found in `mock-events-service.ts`

Reply:
👍 done